### PR TITLE
Fixed jog wheel alignment issue #1898701 in beat sync mode

### PR DESCRIPTION
**Pull request by fkbreitl**
For details see https://bugs.launchpad.net/mixxx/+bug/1898701

### CODE REVIEW

File: Pioneer-DDJ-200-scripts.js [hunk 1]
```diff
@@ -13,18 +13,23 @@ DDJ200.scratch = function (channel, control, value, status, group) {
...
     engine.setValue(group, 'jog', value - 64);
 };
 
+DDJ200.jog = DDJ200.jog1;
+
 DDJ200.touch = function (channel, control, value, status, group) {
     var deckNumber = script.deckFromGroup(group);
     if (value == 0) {
         // disable scratch
+        DDJ200.jog = function () {}; // disable nudging to not prevent alignment
```
Comment (dan-giddins) [inline Pioneer-DDJ-200-scripts.js]: I see what you are trying to do here. I have noticed this problem before, and this is probably the only solution to it.

I like the idea of what you are trying to do. I think that enabling or disabling the nudging could however be done more neatly by setting a boolean in the DDJ200 prop here, and checking this boolean in jog(), as opposed to trying to use a 'jog' and 'jog1'. What do you think?

Comment (dan-giddins) [reply Pioneer-DDJ-200-scripts.js]: Also is 900ms the best time to use? Ideally we want it to be as short as possible, to allow us to nudge again ASAP without it coming off too early such that we get this 'Inertial nudge' effect

**Review by fkbreitl — COMMENTED** [review]
Disable jog using jog_disable variable

**Review by dan-giddins — APPROVED** [review]

File: Pioneer-DDJ-200-scripts.js (hunk 1 above)
Comment (fkbreitl) [reply Pioneer-DDJ-200-scripts.js]: It was the smallest save value.

**Pull request merged**